pkg/utils/syscalls/amd64/syscalls.go

```diff
@@ -0,0 +1,968 @@
+package amd64
+
+// This updated to kernel 6.6-rc2
+var SyscallsNameToNumber = map[string]int{
+	"_llseek":                      -1,
```

Comment:
What does it mean to have `-1`, that the syscalls is not available on this arch? if yes, I think we could remove them.

Reply:
Done